### PR TITLE
Fix Grammar in readme

### PR DESCRIPTION
**Pull request by wrac4242**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,16 +14,16 @@
...
-If you want to contribute, feel free to fork this repository, read the [contributing information](https://github.com/osu-player/osuplayer#-contributing-to-the-project) and head to the [official osu!player discord](https://discord.gg/RJQSc5B) (not mandetory, but helpful). There you can add all valuable ideas and discuss other stuff regarding development.
+If you want to contribute, feel free to fork this repository, read the [contributing information](https://github.com/osu-player/osuplayer#-contributing-to-the-project) and head to the [official osu!player discord](https://discord.gg/RJQSc5B) (not mandatory, but helpful). There you can add all valuable ideas and discuss other stuff regarding development.
 
 ## ☝️ Requirements
 
 #### osu!player requirements
 ✔️ A working computer  
 ✔️ .NET 6 or later installed  
-✔️ osu! installed with an **osu!.db file** or **osu!lazer client.realm** *(Beatmaps imported in osu!)*  
+✔️ osu! installed with an **osu!.db file** or **osu!lazer client.realm** *(Beat maps imported in osu!)*
```
Comment (Founntain) [inline README.md:26]: Beatmaps is one word.

File: README.md [hunk 2]
```diff
@@ -66,17 +66,17 @@ Best practice is, that we discuss (new) features and if they are needed. So don'
...
 - [x] Audio-Equalizer 
 - [ ] Localization 
 - [ ] Language Editor 
 - [ ] Language Manager to upload and download translations   
 - [ ] Hotkey support  
-- [ ] Export songs to directory  
+- [ ] Export songs to the directory  
 - [ ] Custom themes 
-- [ ] Miniplayer to save some space  
+- [ ] Mini player to save some space
```
Comment (Founntain) [inline README.md:79]: One word, because that's how it's called and know by the community. As it is a *name*

**Review by Founntain — CHANGES_REQUESTED** [review]
Looks fine, except of these small things

File: README.md (hunk 1 above)
Comment (Cesan) [inline README.md:26]: "Beatmaps" is a term of osu! and I would rather write this as one word instead of two.

**Review by Cesan — CHANGES_REQUESTED** [review]
See requests. Otherwise thanks for the corrections ^^

File: README.md (hunk 1 above)
Comment (Founntain) [reply README.md:26]: I WAS FASTER HAHAHAHA

Comment (wrac4242) [reply README.md]: resolved

File: README.md (hunk 2 above)
Comment (wrac4242) [reply README.md]: resolved

**Review by Cesan — APPROVED** [review]
Looks good. Thanks for the fixes

**Review by Founntain — APPROVED** [review]

**Pull request merged**